pymoab-visit-py2-18.04

```diff
@@ -0,0 +1,28 @@
+FROM svalinn/pymoab-py2-18.04:latest
+
+ENV HOME /root
+
+RUN apt-get install -y --fix-missing \
+    wget \
+    cpio \
+    libpcre3-dev \
+    libgl1-mesa-glx \
+    libgl1-mesa-dev \
+    libsm6 \
+    libxt6 \
+    libglu1-mesa
+
+RUN pip install xmldiff
+
+RUN cd $HOME/opt \
+    && wget https://github.com/visit-dav/visit/releases/download/v3.1.1/visit3_1_1.linux-x86_64-ubuntu18.tar.gz \
+    && wget https://github.com/visit-dav/visit/releases/download/v3.1.1/visit-install3_1_1 \
+    && echo 1 > input \
+    && bash visit-install3_1_1 3.1.1 linux-x86_64-ubuntu18 /usr/local/visit < input \
+    && rm -rf visit3_1_1.linux-x86_64-ubuntu18.tar.gz
```

Comment:
maybe remove the install script as well ?

Does the install scripts clean after itself ? (remove the sources after build/install)